cirq/ops/projectors.py

```diff
@@ -26,7 +26,7 @@ class Projector(raw_types.Gate):
     """
 
     def __init__(self,
-                 projection_basis: Union[List[List[float]], np.ndarray],
+                 projection_basis: Union[STATE_VECTOR_LIKE],
```

Comment:
These should be a sequence
```suggestion
                 projection_basis: Sequence[STATE_VECTOR_LIKE],
```

Reply:
Done.